tests/irdl/test_declarative_assembly_format.py

```diff
@@ -1559,7 +1562,7 @@ class OptSuccessorOp(IRDLOperation):
 # Inference                                                                   #
 ################################################################################
 
-_T = TypeVar("_T", bound=Attribute)
+_T = TypeVar("_T", bound=Attribute, covariant=True)
```

Comment:
Necessary for constr definition below, that way `BaseAttr[ParamOne[_T]]` is a subclass of `BaseAttr[ParamOne[Attribute]]`.